script/nonEnglishContent.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-console */
+const libxmljs2 = require('libxmljs2');
+const fetch = require('node-fetch');
+
+const SITEMAP_URL = `http://${process.env.WEB_HOST || 'localhost'}:${process.env
+  .WEB_PORT || 3001}/sitemap.xml`;
+const SITEMAP_LOC_NS = 'http://www.sitemaps.org/schemas/sitemap/0.9';
```

Comment:
Trying to understand the use case here...A team member would like an inventory of all translated pages on the website. They get in touch with you. You pull latest CMS content, start your project, then run this script, which produces the correct output, then you copy and paste that output into a Slack message or GitHub comment. Is that that the vision?

Reply:
yes, I think that is the idea. @SKasimow any thoughts here?
Nick, for context, this is the ticket description:
```As a PM on the C+L team, I would like to have a complete inventory of all translated content that is currently on Va.Gov today.
This includes both pages and PDFs.```

Reply:
Hello! Yes. Nick - by way of background, we are going to be creating a new Spanish and Tagalog resource page which will provide users with some useful information and a list of existing in-language content they can link to from the new page. As part of this effort, we'd like to know what is out there today so that we can check-usage, verify that content is up-to-date, know which URL to link and the like. I know this will only address content already in Drupal but it will be a start. So for the output,  would like to have some info in a spreadsheet (if possible) with Page Title, URL, last modified date ---- and anything else that may be useful for vetting the pages (I'm not sure what we can pull).  Does that make sense?

Reply:
Thanks! That makes sense yeah, I understand. I appreciate the clarification. My thinking was - how can we do this in such a way that you as a product manager have access to this information whenever you want it, rather than needing to ask an engineer to pull it for you.

Reply:
@ncksllvn Hi Nick - ah, good thought! I think we'll have a good eye into any new spanish/tagalog content being added as Danielle (or her team) will most likely be involved. The bigger challenge  is going to be tracking down content/pdf's that haven't been migrated to Drupal yet. If you have any ideas, I'm all ears :)

Reply:
Also, do we know if we will be able to get Page Title, URL, last modified date? Anything else that might be useful to grab? cc: @DanielleThierryUSDSVA Thanks!

Reply:
the sitemap provides the url and the last modified, nothing else is provided. you can see the structure here:
https://www.va.gov/sitemap.xml